filebeat/module/haproxy/log/test/tcplog.log

```diff
@@ -1 +1 @@
-Sep 20 15:44:23 127.0.0.1 haproxy[25457]: 127.0.0.1:40962 [20/Sep/2018:15:44:23.285] main app/<NOSRV> -1/-1/0 212 SC 1/1/0/0/0 0/0
+Sep 20 15:44:23 127.0.0.1 haproxy[25457]: 127.0.0.1:40962 [20/Sep/2018:15:44:23.285] main app/<NOSRV> -1/-1/1 212 SC 1/1/0/0/0 0/0
```

Comment:
We should always use at least 2 log lines to make sure things work as expected. Not a big issue here as we don't have multiline.

Reply:
Agreed